### PR TITLE
Enable filtering and pruning by either releases or target issues

### PR DESCRIPTION
**Pull request by str4d**

### CODE REVIEW

File: zcash-issue-dag.py [hunk 1]
```diff
@@ -92,15 +92,15 @@
 
 REPOS = REPO_SETS[DAG_VIEW]
 
-# Whether to remove issues and PRs that are not target issues.
-ONLY_TARGETS = strtobool(os.environ.get('ONLY_TARGETS', 'false'))
+# Whether to remove issues and PRs that are not target or release issues.
+ONLY_INCLUDE = os.environ.get('ONLY_INCLUDE', '')
```
Comment (nuttycom) [inline zcash-issue-dag.py:96]: We can only choose one category, not both?

**Review by nuttycom — COMMENTED** [review]
This looks like it lets you select releases or targets, but not both; it seems like it might be useful to allow both?

File: zcash-issue-dag.py (hunk 1 above)
Comment (str4d) [reply zcash-issue-dag.py:96]: Indeed, that was the fastest thing to get working. I wanted you to test and ensure this did what you wanted, before I went further.

**Review by nuttycom — APPROVED** [review]
utACK

**Review by daira — COMMENTED** [review]
Post-hoc ACK

**Pull request merged**